elated-egrets/bangazon-internal-site-2.0
# category add form and redirect from categories view to the add form added

## DESCRIPTION
Added category add form and redirect to it from the categories view

## How to pull down
Commands

```
git fetch --all
git checkout lsCategoryForm
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. login and go to the categories view
1. click the add category button
1. name the category and submit the form
1. you should be redirected back to the categories view and the newly added category should be visible with 0 listings

## What you changed

## What was added to read me

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Have 2 teammates approved the PR?

Then Ship It!
